<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QWDialogHeaders</class>
 <widget class="QDialog" name="QWDialogHeaders">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>表头标题</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" rowspan="5">
       <widget class="QListView" name="listView"/>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="BtnAddHeader">
        <property name="text">
         <string>添加标题</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="BtnInsetHeader">
        <property name="text">
         <string>插入标题</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="BtnDelHeader">
        <property name="text">
         <string>删除标题</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="BtnOk">
        <property name="text">
         <string>确定</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="BtnCancel">
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
